pkg/kubernetes/run.go

```diff
@@ -81,6 +83,21 @@ func (k *KubernetesDriver) RunDevContainer(
 		options = containerInfo.Options
 	}
 
+	cachePVC, _, err := k.getDevContainerPvc(ctx, cacheClaimName)
```

Comment:
What do you think of exposing a control in the provider.yaml for this behaviour? 
I can imagine not everyone wanting to have a shared cache

Additionally, we probably want to disable this in Pro in favor of an external registry cache. Otherwise every workspace would create it's own cache pvc in addition to the workspace pvc
🤔

Reply:
Ah and I think we only want this for dockerless builds, no?
If we already have an image we can rely on the kubelets caching mechanism

Reply:
What about as a CLI parameter instead? devpod CLI executes this and also assumes a mount point for the cache so we could expose a context option and set it this way. Otherwise devpod CLI could attempt to create a mount point and the provider could fail because of the value in provider.yaml.

I think Pro users would want local and remote cache since local cache is much faster. The cache is actually scoped to the cluster (named "dockerless-cache"), unlike the workspace volume we only have 1

Reply:
I see the point but the problem with putting it into the CLI is that it's only applicable to workspaces created using the kubernetes provider 🤔 

> the cache is actually scoped to the cluster (named "dockerless-cache")
iirc the pvc is namespace bound so there's a good chance you don't have access to it if you deploy to a separate namespace 🤔